ISPCPromWeb/myvet
# feat: 13 - Shopping cart

### What does this PR do?

With this PR we are adding Shopping Cart, PLP and PDP pages, and Store, Cart, Product components.

### What is changing?

- Components
- Pages
- Layouts

### How should it be tested manually?

- Go to `/store ` and validate the visibility of the products and their name, formatted price, image and description 
- Go to any PDP page and validate the products name, formatted price, image and description visibility
- Go to `/Cart` and validate it's client and purchase form, and the cart.

**Expected result:** You should be able to see he products and their attributes

### Which are the relevant tickets?

- #13

### Are there any related PRs?

- N/A

### Screenshots

- N/A
